neurom/core/tests/test_soma.py

```diff
@@ -149,20 +140,20 @@ def test_Soma_Cylinders():
     assert_array_equal(s.center, [0, 0, -10])
     nt.ok_('SomaCylinders' in str(s))
 
-    # cylinder: h = 10, r = 20
-    s = load_neuron(StringIO(u"""
-                1 1 0   0 0 20 -1
-                2 1 0 -10 0 20  1"""), reader='swc').soma
+    # # cylinder: h = 10, r = 20
```

Comment:
why area these test commented and not removed?

Reply:
Benoit commented them out, so tests will pass. I don't know what to do here. Previously these somas were loaded as Cylinder ones. Now they are loaded as Contour ones. Hence they don't have areas. I will remove it then.

Reply:
SWC files should be cylinders, I wonder what is happening?

Reply:
According to MorphIO this is incorrect spec for cylinder soma. It should be either one or three points, not two.

Reply:
A cylinder soma can be as many points as one wants; there are just 2 special case: the 1 point implies a 'sphere', and the 3 point case where the points aren't in their natural order w/ the +/- diameter.

What is the error MorphIO is raising?

Reply:
MorphIO does not raise anything. It only considers this soma as Contour one:

```python
from io import StringIO

s = load_neuron(StringIO(u"""
            1 1 0   0 0 20 -1
            2 1 0 -10 0 20  1"""), reader='swc')
print(s.soma.points)
print(type(s.soma))
```
```
[[  0.   0.   0.  20.]
 [  0. -10.   0.  20.]]
<class 'neurom.core._soma.SomaSimpleContour'>
```

Reply:
Thanks. That seems incorrect on the MorphIO side; I have opened a issue: https://github.com/BlueBrain/MorphIO/issues/265